tests/client.presence.sentry.test.js

```diff
@@ -78,8 +82,12 @@ describe('given a client and a server,', function() {
           done();
         };
 
-        publish_client_online(p.client);
-        p.on(4, validate);
+        publishClientOnline(p.client);
+        setTimeout(function() {
+          sentry.stop();
```

Comment:
I am very surprised how this works. The idea of this test is, "a sentry died long ago". But here it seems we just stopped it a little while back. How is this working?